### PR TITLE
correct symbols(::PolyRing) and do away with singular's native printing

### PR DESCRIPTION
**Pull request by tthsqe12**

### CODE REVIEW

File: src/number/n_algExt.jl [hunk 1]
```diff
@@ -120,11 +120,12 @@ function AbstractAlgebra.expressify(n::n_algExt; context = nothing)::Any
    end
 end
 
-function show(io::IO, n::n_algExt)
-   libSingular.StringSetS("")
-   libSingular.n_Write(n.ptr, parent(n).ptr, false)
-   m = libSingular.StringEndS()
-   print(io, m)
+function Base.show(io::IO, ::MIME"text/plain", a::n_algExt)
```
Comment (rfourquet) [inline src/number/n_algExt.jl:123]: This 3-arg method normally defaults to calling the 2-arg one (without MIME), can't this method be therefore deleted here?

Comment (tthsqe12) [reply src/number/n_algExt.jl]: ok

File: test/number/n_algExt-test.jl [hunk 2]
```diff
@@ -22,18 +22,19 @@ end
...
    p = (3*a+2)*x+(2*a^2)*y+a*z
    q = (a+1)*x^2
-   # singular's printing
-   @test string(p) == "(3*a+2)*x-2*y+(a)*z"
-   @test string(q) == "(a+1)*x^2"
-   # AA's printing
-   @test sprint(show, "text/plain", p) == "(3*a + 2)*x - 2*y + a*z"
-   @test sprint(show, "text/plain", q) == "(a + 1)*x^2"
+
+   @test sprint(show, "text/plain", p) ==
```
Comment (thofma) [inline test/number/n_algExt-test.jl:33]: Why the newline?

Comment (tthsqe12) [reply test/number/n_algExt-test.jl]: ok

**Pull request merged**